File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Features/FormBorderStylesPage.vb
Namespace CompanionKit.Controls.Form.Features

	Public Class FormBorderStylesPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

            ' Fill up with the form border styles.
            Me.mobjComboBox.DataSource = Global.System.Enum.GetValues(GetType(BorderStyle))


		End Sub

        ''' <summary>
        ''' Handles Click event of the button.
        ''' Opens Form with the specified border style.
        ''' </summary>
        Private Sub mobjButton_Click(sender As Object, e As EventArgs) Handles mobjButton.Click
            Dim form As New BorderStyleForm()
            form.BorderStyle = DirectCast(Me.mobjComboBox.SelectedItem, Gizmox.WebGUI.Forms.BorderStyle)
            form.ShowDialog()
        End Sub
	End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Features/FormBorderStylesPage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.Form.Features
{
    public partial class FormBorderStylesPage : UserControl
    {
        public FormBorderStylesPage()
        {
            InitializeComponent();
            
            // Fill up with the form border styles.
            this.mobjComboBox.DataSource = Enum.GetValues(typeof(global::Gizmox.WebGUI.Forms.BorderStyle));
        }

        /// <summary>
        /// Handles Click event of the button.
        /// Opens Form with the specified border style.
        /// </summary>
        private void mobjButton_Click(object sender, EventArgs e)
        {
            BorderStyleForm form = new BorderStyleForm();
            form.BorderStyle = (Gizmox.WebGUI.Forms.BorderStyle)this.mobjComboBox.SelectedItem;
            form.ShowDialog();
        }
    }
}